yo1k/tic_tac_toe/core/game.py

```diff
@@ -0,0 +1,296 @@
+from enum import Enum, auto
+from collections.abc import Sequence
+from typing import Optional
+from abc import ABC, abstractmethod
+
+
+class Mark(Enum):
+    X = auto()
+    O = auto()
+
+
+class Player:
+    def __init__(self, mark: Mark):
+        self.__mark = mark
+        self.__wins = 0
+
+    def mark(self) -> Mark:
+        return self.__mark
+
+    @property
+    def wins(self) -> int:
+        return self.__wins
+
+    @wins.setter
+    def wins(self, wins: int):
+        self.__wins = wins
```

Comment:
These public methods simply open up `Player.wins` for read and write actions initiated even from other modules. We actually need module-private write access and public read access to this attribute. I.e., the `@property wins` should stay, while `@wins.setter wins` has to be module-private, which is likely not possible. In that case, it may be replaced with an `_incrementWins` module-private method.

Reply:
Done.
I have made all attributes and methods public of all class objects constituting the `State` class.